.eslintrc.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
```

Comment:
Can we put this in package.json instead of another dotfile?

Reply:
eg https://github.com/OriginProtocol/identity-playground/blob/master/package.json#L105

Reply:
done